integration-cli/docker_cli_run_test.go

```diff
@@ -404,8 +446,15 @@ func (s *DockerSuite) TestRunVolumesFromSymlinkPath(c *check.C) {
 }
 
 func (s *DockerSuite) TestRunExitCode(c *check.C) {
-	testRequires(c, DaemonIsLinux)
-	_, exit, err := dockerCmdWithError("run", "busybox", "/bin/sh", "-c", "exit 72")
+	var (
+		exit int
+		err  error
+	)
+	if daemonPlatform == "windows" {
+		_, exit, err = dockerCmdWithError("run", WindowsBaseImage, "cmd", "/s", "/c", "exit 72")
```

Comment:
No - it appears to be a bug in the busybox image I found. Our new hire is looking at other bugs in it and making sure there's a 64-bit image available for nanoserver testing. This is a workaround though.